assets/RNAseq_report.Rmd

```diff
@@ -1140,7 +1167,7 @@ if (isProvided(params$path_contrast_matrix)){
         write.table(DE_genes_contrast_genename, file=paste("differential_gene_expression/allgenes/DE_contrast_",contname,".tsv",sep=""), sep="\t", quote=F, col.names = T, row.names = F)
 
         # Select only significantly DE
-        DE_genes_contrast <- subset(DE_genes_contrast_genename, padj < params$pval_threshold & abs(log2FoldChange) > params$logFC_threshold)
+        DE_genes_contrast <- subset(DE_genes_contrast_genename, padj <= params$pval_threshold & abs(log2FoldChange) >= params$logFC_threshold)
```

Comment:
Could the parameter `pval_threshold` be renamed to `adj.pval_threshold`?

Reply:
I changed it to `adj_pval_threshold` just to be on the safe side with dots